test/unit/middlewares/authorizeBot.test.js

```diff
@@ -115,4 +116,116 @@ describe("Middleware | Authorize Bot", function () {
       expect(nextSpy.calledOnce).to.be.equal(true);
     });
   });
+
+  describe("Check authorization of bot for discord service", function () {
+    it("should return unauthorized when token is expired or malformed for discord service", function () {
+      const jwtStub = sinon.stub(jwt, "verify").throws(new Error("invalid token"));
+
+      const request = {
+        headers: {
+          authorization: `Bearer ${BAD_TOKEN}`,
+          "x-service-name": DISCORD_SERVICE,
+        },
+      };
+
+      const response = {
+        boom: {
+          badRequest: sinon.spy(),
+          unauthorized: sinon.spy(),
+        },
+      };
+
+      const nextSpy = sinon.spy();
```

Comment:
```suggestion
      const nextSpy = sinon.spy();
```
This has been used in multiple places. Can we write in describe block?

Reply:
done